scripts/split-e2e-tests.ts

```diff
@@ -216,8 +220,9 @@ function splitTests(
 
   const newJobs = testSuites.reduce((acc, suite, index) => {
     const newJobName = generateJobName(jobName, suite);
-    const testRegion = FORCE_US_WEST_2.some(job => newJobName.startsWith(job))
-      ? 'us-west-2'
+    const forceRegion = Object.keys(FORCE_REGION).find(key => newJobName.startsWith(key));
+    const testRegion = forceRegion
+      ? FORCE_REGION[forceRegion as FORCE_TESTS]
```

Comment:
these changes are new to me. Any context here?

Reply:
This change forces the containers test to run in us-east-1 region.

`containers` feature is not supported in all the regions. We officially launched it for us-east-1 only but there are other regions the feature is available now. Forcing the test to use the supported region fix test flakiness.